zqd/ingest/logtailer_test.go

```diff
@@ -0,0 +1,155 @@
+package ingest
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/brimsec/zq/driver"
+	"github.com/brimsec/zq/zio"
+	"github.com/brimsec/zq/zio/tzngio"
+	"github.com/brimsec/zq/zng/resolver"
+	"github.com/brimsec/zq/zql"
+	"github.com/stretchr/testify/suite"
+)
+
+var sortTs = zql.MustParseProc("sort ts")
+
+const expected = `#0:record[ts:time]
+0:[0;]
+0:[1;]
+0:[2;]
+0:[3;]
+0:[4;]
+0:[5;]
+0:[6;]
+0:[7;]
+0:[8;]
+0:[9;]
+0:[10;]
+0:[11;]
+0:[12;]
+0:[13;]
+0:[14;]
+0:[15;]
+0:[16;]
+0:[17;]
+0:[18;]
+0:[19;]
+`
+
+type logTailerTSuite struct {
+	suite.Suite
+	dir  string
+	zctx *resolver.Context
+	dr   *logTailer
+}
+
+func TestLogTailer(t *testing.T) {
```

Comment:
Just want to register that I'm +1 on using testify/suite.

Reply:
It is definitely very useful for certain circumstances.